### DISCUSSION TITLE
[@types/react-transition-group] Why is there [prop: string]: any; in Transition props?

### DISCUSSION THREAD

**Discussion by std4453** [DefinitelyTyped/DefinitelyTyped#55574] [Issues with a @types package]
This does not make sense.
https://github.com/DefinitelyTyped/DefinitelyTyped/blob/5bf630cf14893c1fec8224ffbeb339d59bc9ecd7/types/react-transition-group/Transition.d.ts#L134

[a comment by an automated account is not shown]

**Comment by ybiquitous** [comment]
@std4453 I agree. The code was added via #17824, but I don't know why. 🤷🏼 

https://github.com/DefinitelyTyped/DefinitelyTyped/pull/17824/files#diff-eccec9d3cff135b71befc4b56d309514bcebd1a374a06b805d18cf1fa1ee4ba8R25
